src/platforms/javascript/guides/nextjs/manual-setup.mdx

```diff
@@ -79,67 +79,45 @@ In serverless deployment environments, including Vercel, the Next.js server runs
 To capture these errors in Sentry, you can use the Next.js [error page customization](https://nextjs.org/docs/advanced-features/custom-error-page#reusing-the-built-in-error-page) option. To do this, create `pages/_error.js`, and include the following:
 
 ```javascript {filename:pages/_error.js}
-import NextErrorComponent from "next/error";
+/**
+ * NOTE: This requires `@sentry/nextjs` version 7.3.0 or higher.
+ *
+ * NOTE: If using this with `next` version 12.2.0 or lower, uncomment the
+ * penultimate line in `CustomErrorComponent`.
```

Comment:
I would ditch this note. 1. We already have it down below. 2. I had to google penultimate and others might to - but if we begin writing "delete the line second to last in blabla" it gets a bit ridiculous.